components/ResourceCreation/PatientCreation.tsx

```diff
@@ -34,6 +45,46 @@ function PatientCreation({
   const measurementPeriod = useRecoilValue(measurementPeriodState);
   const [selectedPatient, setSelectedPatient] = useRecoilState(selectedPatientState);
   const [isConfirmationModalOpen, setIsConfirmationModalOpen] = useState(false);
+  const measureBundle = useRecoilValue(measureBundleState);
+  const [showCalculation, setShowCalculation] = useState(false);
+  const { classes } = useStyles();
+
+  // Function to wrap the calculate function and catch errors in fqm-execution
+  const clickCalculateButton = async (id: string | null) => {
+    try {
+      await calculate(id);
+    } catch (error) {
+      console.log(error);
```

Comment:
Can we do something better than just silently console log an error here? Ideally the user can become aware if there are any issues with calculation for a patient.

We already have precedent in this codebase for showing notifications/have the infrastructure set up for it. Can we make use of `showNotification` here instead to have the error message pop up on the screen? See the ImportModal component for examples of where this is done